WordPress/src/main/java/org/wordpress/android/ui/stats/StatsGeoviewsFragment.java

```diff
@@ -219,7 +219,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
             holder.totalsTextView.setText(FormatUtils.formatDecimal(total));
 
             // image (country flag)
-            holder.networkImageView.setImageUrl(PhotonUtils.fixAvatar(imageUrl, mResourceVars.headerAvatarSizePx), WPNetworkImageView.ImageType.BLAVATAR);
+            holder.networkImageView.setImageUrl(GravatarUtils.fixGravatarUrl(imageUrl, mResourceVars.headerAvatarSizePx), WPNetworkImageView.ImageType.BLAVATAR);
```

Comment:
I think here is better to call GravatarUtils.fixGravatarUrl with the default picture set to STATUS_404.